test/helper.js

```diff
@@ -78,6 +78,10 @@ require('abortcontroller-polyfill/dist/polyfill-patch-fetch')
 // localStorage
 window.localStorage = {}
 
+if (!window.localStorage.removeItem) {
+  window.localStorage.removeItem = () => null
+}
+
```

Comment:
Nit: I think we can be fairly certain this is falsy, considering that it's set to an empty object here.

Maybe something like this would make more sense?

```suggestion
window.localStorage = {
  removeItem: () => null
}
```

Reply:
Or you could move this to the single test that requires it in `migrator-test.js`. The simpler this test setup is, the better.

Reply:
@Gudahtt this is actually required by the migration 50 test as well, because
![Screenshot from 2020-12-03 17-34-16](https://user-images.githubusercontent.com/7499938/101088147-cb282a00-358d-11eb-8476-fa6b3b24ccc1.png)

Reply:
The one _other_ case I meant; of course it's needed for this one :sweat_smile: